engine/common/bothdefs.h

```diff
@@ -288,12 +288,11 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 	#undef VOICECHAT	//too lazy to compile opus
 	#undef HLCLIENT		//dlls...
 	#undef HLSERVER		//dlls...
-//	#undef CL_MASTER	//bah. use the site to specify the servers.
+	#undef CL_MASTER	//bah. use the site to specify the servers.
```

Comment:
With CL_MASTER defined, some Master_ func was not declared, given the comment I went with just disabling it again.